### PR TITLE
Changing button component "background" declaration to "background-color".

### PR DESCRIPTION
**Pull request by ghost**
In the spirit of the following article:
http://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/

### CODE REVIEW

**Comment by LeBenLeBen** [comment]
Hey, thanks for the contribution!

As you can see later in the file, I did use `background-color` mostly. In the case of `btn` I see it more like a reset, where we want to uniform buttons and prevent any previous style to apply. For example you have set some `background-image` on an `a` this would influence the style of an `a.btn` if we don’t reset the style with `.btn`.

I’m not totally convinced myself this is better than your proposal but at the moment I think it’s more _secure_ like it is. Do you have any experience you want to share where the current code prevented you from doing something to change my mind?

**Comment by ghost** [comment]
Hi, sorry for the late response.

TL;DR:

Now I too find the `background` shorthand justified; I might have been misled (on the project where I had my issue) by a likely improper extending of the `%btn` placeholder.

Not TL;DR:

Now that I look at it, your reasoning makes all the sense, I would also reset a button initially with a `background` shorthand, since one doesn't want to keep track of how browsers "hack" buttons to end up with their appearance.

The reason I sent the pull request was that I once (it was more than half a year ago) was however burned by this background: on hover, or some similar interaction, a background image that I defined for a button was overridden by this shorthand (or so I recall).

The thing I suspect nevertheless - now that I consider that the selector contains a `%btn` placeholder too - is that this placeholder might have been extended in such ways that resulting selector specificities created unwanted overrides. If indeed this happened, then there was some improper SCSS authoring going on, and the `background` shorthand was not to blame. :+1:
